docs/tutorials/11_GMPHDTutorial.py

```diff
@@ -0,0 +1,590 @@
+    for truth in truths_by_time[i]:
+        new_truths.append([truth.state_vector[0], truth.state_vector[2]])
+    for state in tracks_by_time[i]:
+        new_tracks.append([state.state_vector[0], state.state_vector[2]])
+    for measurement in all_measurements[i]:
+        if type(measurement) == TrueDetection:
+            new_measurements.append([measurement.state_vector[0], measurement.state_vector[1]])
+        elif type(measurement == Clutter):
+            new_clutter.append([measurement.state_vector[0], measurement.state_vector[1]])
+
+    # Plot the contents of these lists on the right axis
+    if len(new_truths) > 0: truths.set_offsets(new_truths)
+    if len(new_tracks) > 0: tracks.set_offsets(new_tracks)
+    if len(new_measurements) > 0: measurements.set_offsets(new_measurements)
+    if len(new_clutter) > 0: clutter.set_offsets(new_clutter)
+
+    # Create a legend. The use of Line2D is purely for the visual in the legend
+    data_types = [Line2D([0], [0], color='white', marker='o', markerfacecolor='blue', markersize=15,
+                         label='Ground Truth'),
+                 Line2D([0], [0], color='white', marker='o', markerfacecolor='orange', markersize=15,
+                         label='Clutter'),
+                 Line2D([0], [0], color='white', marker='o', markerfacecolor='green', markersize=15,
+                         label='Detection'),
+                 Line2D([0], [0], color='white', marker='o', markerfacecolor='red', markersize=15,
+                         label='Track')]
+    axR.legend(handles=data_types)
+
+    return sf, truths, tracks, measurements, clutter
+
+# Set up the x, y, and z space for the 3D axis (the left one)
+xx = np.linspace(x_min-5, x_max+5, 100)
+yy = np.linspace(y_min-5, y_max+5, 100)
+x, y = np.meshgrid(xx, yy)
+zarray = np.zeros((100, 100, number_steps))
+
+# Create the matplotlib figure and axes.
+fig = plt.figure(figsize=(10, 6))
+axL = fig.add_subplot(121, projection='3d')
+axR = fig.add_subplot(122)
+axR.set_xlim(x_min-5, x_max+5)
+axR.set_ylim(y_min-5, y_max+5)
+
+# Add an initial surface to the left axis and scattered points on the right axis. Doing
+# this now means that in the animate() function we only have to update these variables
+sf = axL.plot_surface(x, y, zarray[:, :, 0], cmap=cm.RdBu, linewidth=0, antialiased=False)
+truths = axR.scatter(x_min-10, y_min-10, c='blue', linewidth=6, zorder=0.5)
+tracks = axR.scatter(x_min-10, y_min-10, c='red', linewidth=4, zorder=1)
+measurements = axR.scatter(x_min-10, y_min-10, c='green', linewidth=4, zorder=0.5)
+clutter = axR.scatter(x_min-10, y_min-10, c='orange', linewidth=4, zorder=0.5)
+
+# Create and display the animation
+from matplotlib import rc
+anim = animation.FuncAnimation(fig, animate, frames=number_steps, interval=500,
+                               fargs = (sf, truths, tracks, measurements, clutter), blit=False)
+rc('animation', html='jshtml')
+anim
+
```

Comment:
```suggestion

# sphinx_gallery_thumbnail_number = 4
```

I think this will make your last plot/animation will be the thumbnail on the tutorials home page. If it doesn't work it might be because it's an animation (I'm not sure how it works for those), in which case try moving it to after the previous plot and change the number to 3 instead.

Reply:
Ok, thank you! I will commit this suggestion and try building the documentation locally. If it does not work then I will change it to 3.

Reply:
Sorry for the late update. In the notebook itself there are 2 plots being animated side by side. Unfortunately, the thumbnail only showed the right side plot being animated. The one on the right is a 2D plot of the space, but without the corresponding 3D Gaussian mixtures being animated it didn't make sense and looked a little weird. I think Sphinx made this mistake because the animation figure has two axes which are plotted separately. I changed it to thumbnail_number=3, showing the ground truth, measurements, and tracks. I think this looks better but let me know if you would like me to change it back! I've attached files showing the animation thumbnail (number=4) and the plot thumbnail (number=3) for reference.

https://user-images.githubusercontent.com/59794482/125655333-7a816baf-8ae8-4b7b-bacc-1fffd383e3ca.mov

<img width="276" alt="Screen Shot 2021-07-14 at 12 08 02 PM" src="https://user-images.githubusercontent.com/59794482/125655339-9ed52af0-20d8-4e45-b3f6-3c9ee3e2ac4a.png">